### ISSUE TITLE
Track Head Training

### ISSUE THREAD

**Issue by chicleee** [facebookresearch/vggt#261]
Thanks you for your great work!

I'm trying to train the track head, but I'm not sure what **query_points** should be during the training. Should it be batch["tracks"] or keypoints from the extractor?

Looking forward to your reply :)

**Comment by chicleee** [comment]
Now I think it should be batch["tracks"][:, 0, :, :], right?

But I'm not sure what the following code does in the dirty track loss, which has caused a shape mismatch in sequence_loss. 
`    
# if self.training and hasattr(self, "train_query_points"):
    train_query_points = coord_preds[-1].shape[2]
    gt_tracks = gt_tracks[:, :, :train_query_points]
    gt_tracks = check_and_fix_inf_nan(gt_tracks, "gt_tracks", hard_max=None)
    gt_track_vis_mask = gt_track_vis_mask[:, :, :train_query_points]
`

Commenting it out seems to work, though.
